src/Commands/Self/ConfigDumpCommand.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Self;
+
+use Consolidation\OutputFormatters\StructuredData\PropertyList;
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Robo\Contract\ConfigAwareInterface;
+
+class ConfigDumpCommand extends TerminusCommand
+{
+    /**
+     * Display the current Terminus configuration information
+     *
+     * @command self:config:dump
+     *
+     * @return RowsOfFields
+     *
+     * @usage terminus self:config-dump
+     *    Show the current configuration values
+     */
+    public function dumpConfig()
+    {
+        $out = [];
+        $config = $this->getConfig();
+        foreach ($config->keys() as $key) {
+            $out[] = [
+                'key' => $key,
+                'env' => $config->getConstantFromKey($key),
+                'value' => $config->get($key),
+                'source' => $config->getSource($key),
+            ];
+        }
+        return new RowsOfFields($out);
```

Comment:
It's weird; the values on this don't output as a table, but a list. Is that what we want? I'm finding a bit troublesome to scroll through the list.
![image](https://cloud.githubusercontent.com/assets/868204/20853187/400a4f44-b89f-11e6-9ed1-f47122d9970b.png)